tests/prompt/test_vc.py

```diff
@@ -0,0 +1,130 @@
+import os
```

Comment:
okay i juggled the commits around and this way should preserve git history. github isn't really showing it, but you can see the git history if you click "View file" and then "Blame", like [here](https://github.com/xonsh/xonsh/blame/52338336cc369be3904bd2927ef129f0dad61410/tests/prompt/test_vc.py)